### ISSUE TITLE
Handle system driver management

### ISSUE THREAD

**Issue by harisvsulaiman** [elementary/appcenter#661]
This project from solus seems like a good starting point.
https://github.com/solus-project/linux-driver-management

**Comment by peteruithoven** [comment]
Would that be a better solution than [fwupd](https://fwupd.org/)? (See issue: https://github.com/elementary/appcenter/issues/498)

**Comment by Conan-Kudo** [comment]
No. `fwupd` and `linux-driver-management` handle different areas of concern.
